src/modules/cpu/kernel/brightness.hpp

```diff
@@ -149,7 +166,15 @@ RppStatus brightness_u8_u8_host_tensor(Rpp8u *srcPtr,
                 int vectorLoopCount = 0;
                 for (; vectorLoopCount < alignedLength; vectorLoopCount += vectorIncrementPerChannel)
                 {
-#if __AVX2__
+#if __AVX512__
+                    __m512 p[6];
+                    rpp_simd_load(rpp_load96_u8pln3_to_f32pln3_avx512, srcPtrTempR,
```

Comment:
vectorIncrementPerChannel has been set as 16 irrespective of AVX512/AVX2/SSE
Here it is not 16 values per channel we are loading, it is 32 per channel if i am not wrong

Please check once and update the vectorIncrementPerChannel accordingly for this variation

Reply:
Done